recipes/spaln/patch-makefile_ldflags

```diff
@@ -0,0 +1,20 @@
+--- src/Makefile.in.orig	2019-04-11 13:29:08.707327000 +0200
++++ src/Makefile.in	2019-04-11 13:44:56.235326000 +0200
+@@ -76,13 +76,13 @@
+ 	ls $(SRC) $(HDR)
+ 
+ spaln:	spaln.o blksrc.o $(SLIB)
+-	$(CXX) -o spaln spaln.o blksrc.o $(SLIB) $(LLIB)
++	$(CXX) $(LDFLAGS) -o spaln spaln.o blksrc.o $(SLIB) $(LLIB)
+ sortgrcd:	sortgrcd.c $(SLIB)
+-	$(CXX) -o sortgrcd sortgrcd.c $(SLIB) $(LLIB)
++	$(CXX) $(LDFLAGS) -o sortgrcd sortgrcd.c $(SLIB) $(LLIB)
```

Comment:
Is it possible to add a test for one of these, just to ensure they're getting built correctly?

Reply:
Anyway, the changes otherwise look good, :+1: from me!